src/modules/vms/infrastructure/repositories/__tests__/vm.typeorm.repository.spec.ts

```
@@ -0,0 +1,70 @@
+import { Test } from '@nestjs/testing';
+import { DataSource } from 'typeorm';
+import { VmTypeormRepository } from '../vm.typeorm.repository';
+import { Vm } from '../../../domain/entities/vm.entity';
```

ESLint

[error] 'Vm' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)